Scenes/escape menu/escape menu.gd

```diff
@@ -0,0 +1,25 @@
+extends Control
+
+var is_paused = false
+
+func _unhandled_input(event):
+	if event.is_action_pressed("pause"):
+		self.is_paused = !is_paused
+		get_tree().paused = is_paused
```

Comment:
The `self.` on line 7 is confusing me. Should we always be referencing with `self`?

```
self.is_paused = !self.is_paused
get_tree().paused = self.is_paused
```

Reply:
oh, I was trying to use setget but turns out how that works changed between godot3 and 4, using self was supposed to stop setget from doing some infinite recursion but that's no longer the case. After abandoning the setget thing I just didn't think to remove it.

![image](https://github.com/ludumipsum/ld55/assets/9700853/9c07b5b1-960e-4b02-9b6d-7cf0af7292a8)

Reply:
I just removed them